src/components/Main.css

```diff
@@ -20,7 +19,6 @@
   background-color: #fff;
   border-top: 1px solid #e5e5e5;
   border-bottom: 1px solid #e5e5e5;
-  z-index: 10;
```

Comment:
Was this property redundant before? did it not have an effect?

Reply:
@GusBaamonde You can get me out of the doubt, before the grid implementation this component needed the z-index for show the header and footer but using Grid and z-index now that is unnecessary?

Reply:
I will do rollback of this change and I will check what happen with the z-index of our modal.

Reply:
Done 🚀